server/game/Card.js

```diff
@@ -58,21 +58,9 @@ class Card extends PlayableObject {
         this.childCards = [];
         this.clonedNeighbors = null;
 
-        this.printedAttack = cardData.attack
-            ? typeof cardData.attack === 'string'
-                ? 0
-                : cardData.attack
-            : 0;
-        this.printedLife = cardData.life
-            ? typeof cardData.life === 'string'
-                ? 0
-                : cardData.life
-            : 0;
-        this.printedRecover = cardData.recover
-            ? typeof cardData.recover === 'string'
-                ? 0
-                : cardData.recover
-            : 0;
+        this.printedAttack = cardData.attack || 0;
+        this.printedLife = cardData.life || 0;
+        this.printedRecover = cardData.recover || 0;
```

Comment:
i looked around and i think this is safe (in combo with the change down below). Otherwise it's too easy to accidentally use printedAttack thinking X is 0